### PR TITLE
release 0.13.42

### PR DESCRIPTION
**Pull request by robin-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/release-notes.rst [hunk 1]
```diff
@@ -6,6 +6,82 @@ Release notes
...
+DAML-ON-X-SERVER
+~~~~~~~~~~~~~~~~
+
+- Made ledger api server to bind to localhost by default instead to the public
+  interface for security reasons.
+
+DAML Assistant
+~~~~~~~~~~~~~~
+
+- Bash completions for the DAML assistant are now available via ``daml install``. These will be
```
Comment (cocreature) [inline docs/source/support/release-notes.rst]: This is missing the ZSH completions which have been merged in the meantime.

Comment (robin-da) [reply docs/source/support/release-notes.rst]: Is it enough to just add `Bash/Zsh`? Seems there was no change log about this.

Comment (cocreature) [reply docs/source/support/release-notes.rst]: There is a changelog but it was probably merged after you generated this https://github.com/digital-asset/daml/commit/d1449e78c7faa80bdaf5496f3585bbd8c44d4cba

Comment (robin-da) [reply docs/source/support/release-notes.rst]: ok, rebased and included.

File: docs/source/support/release-notes.rst [hunk 2]
```diff
@@ -6,6 +6,86 @@ Release notes
...
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Rename ``argument`` in active contract to ``payload``. See #3826.
+- Change variant JSON encoding. The new format is ``{ tag: data-constructor, value: argument }``.
+  For example, if we have: ``data Foo = Bar Int | Baz``, these are all valid JSON encodings for
+  values of type Foo:
+  - ``{"tag": "Bar", "value": 42}``
+  - ``{"tag": "Baz", "value": {}}``
+  See #3622
+- Fix ``/contracts/lookup` find by contract key.
```
Comment (cocreature) [inline docs/source/support/release-notes.rst]:
```suggestion
- Fix ``/contracts/lookup`` find by contract key.
```

File: docs/source/support/release-notes.rst [hunk 3]
```diff
@@ -6,6 +6,86 @@ Release notes
...
+
+
+DAML Compiler
+~~~~~~~~~~~~~
+
+- Move more types from daml-stdlib to standalone LF packages. The module names for the types have
+  also changed slightly. This only matters over the Ledger API when you specify the module name
+  explicitly. In DAML you should continue to use the existing module names.
+
+  - The types from ``DA.Semigroup` are now in a separate package under
```
Comment (cocreature) [inline docs/source/support/release-notes.rst]:
```suggestion
  - The types from ``DA.Semigroup`` are now in a separate package under
```

File: docs/source/support/release-notes.rst [hunk 4]
```diff
@@ -6,6 +6,86 @@ Release notes
...
+  ``DA.Semigroup.Types``
+  - The types from ``DA.Monoid` are now in a separate package under
```
Comment (cocreature) [inline docs/source/support/release-notes.rst]:
```suggestion
  - The types from ``DA.Monoid`` are now in a separate package under
```

**Comment by bame-da** [comment]
Rst is layout aware, like DAML :)
A bullet starts a new block so you need to indent additional likes accordingly.

**Comment by robin-da** [comment]
All manual tests have passed.

**Pull request merged**